psa-arch-tests/api-tests/CMakeLists.txt

```diff
@@ -582,14 +582,18 @@ add_custom_target(
         COMMAND ${CMAKE_COMMAND} --build ${CMAKE_CURRENT_BINARY_DIR}/src/${PSA_TARGET_GENERATE_DATABASE}-build/ -- clean
 )
 
+if(NOT DEFINED PSA_TOOLCHAIN_FILE)
+  set(PSA_TOOLCHAIN_FILE ${PSA_ROOT_DIR}/tools/cmake/compiler/${TOOLCHAIN}.cmake)
+endif()
+
 # Check for supported toolchain/s
 if(${TOOLCHAIN} IN_LIST PSA_TOOLCHAIN_SUPPORT)
 	if (DEFINED CROSS_COMPILE)
 		if(NOT (${TOOLCHAIN} IN_LIST CROSS_COMPILE_TOOLCHAIN_SUPPORT))
 			message(FATAL_ERROR "[PSA] : Error: CROSS_COMPILE not supported for this toolchain, supported toolchain are : ${CROSS_COMPILE_TOOLCHAIN_SUPPORT}")
 		endif()
 	endif()
-	include(${PSA_ROOT_DIR}/tools/cmake/compiler/${TOOLCHAIN}.cmake)
+    include(${PSA_TOOLCHAIN_FILE})
```

Comment:
why the extra indentation here ?
It doesn't align with the `endif()` above.

Reply:
Looks like I need to configure my editor, it was showing the same indentation.

Will fix.

I see that this project uses a mix of tabs and spaces for indentation, whereas Zephyr uses only whitespace.

Not sure how to configure emacs for a mix of tabs and spaces ...

Reply:
![cropped](https://user-images.githubusercontent.com/609230/131630078-f39070db-3832-45b1-88de-a586871bcf74.png)

Reply:
I guess I could edit in whitespace-mode and ensure I use the same blend of tabs and whitespaces as the line above.

Reply:
... or we could accept that upstream does not have sane management of whitespace and accept the natural indentation issues that follow.

Reply:
> I see that this project uses a mix of tabs and spaces for indentation, whereas Zephyr uses only whitespace.

😞 not much we can do then, we shouldn't cleanup their mess.
Of course we should probably just keep the existing indentation when modifying an already existing line, but then it still becomes tricky if we add new lines.

Will leave the choice up to you, as regardless of what we do, it will be a mess.

Thanks for looking into this.